### ISSUE TITLE
Stop/pause music that is playing in background when playing with flutter_sound

### ISSUE THREAD

**Issue by nomyzs** [dooboolab/flutter_sound#219]
### Version of flutter_sound
2.0.4

## flutter doctor
```
[✓] Flutter (Channel stable, v1.12.13+hotfix.7, on Mac OS X 10.14.6 18G2022, locale en-PL)
    • Flutter version 1.12.13+hotfix.7 at /Users/szymonjarosz/work/development/flutter
    • Framework revision 9f5ff2306b (2 weeks ago), 2020-01-26 22:38:26 -0800
    • Engine revision a67792536c
    • Dart version 2.7.0

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.0)
    • Android SDK at /Users/szymonjarosz/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.1, Build version 11A1027
    • CocoaPods version 1.8.4

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 42.1.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
```

### Platforms you faced the error (IOS or Android or both?)
Android.

### Expected behavior
Pause/stop music (i.e. spotify, soundcloud) on Android when playing sounds with flutter_sound

### Actual behavior
Music doesn't stop on Android

### Tested environment (Emulator? Real Device?)
Real device, Pixel 3a

### Steps to reproduce the behavior

1. Play music in soundcloud or spotify in the background
2. play sound with flutter_sound

**Comment by hyochan** [comment]
We should verify that stopping all other sounds running in the background is possible in current app. If I remember correctly, it would be possible but I have to go research on this or someone else can go for research!

**Comment by Larpoux** [comment]
The next flutter_sound release (flauto) adds a great future : 
The developer can parameter its audio session (for example to duck-others/stop or play simultaneously Spotify ...
It can also control its audio-focus.
Spotify volume can be lowered, stopped, or can play along with flutter_sound.

Unfortunately flauto is actually not ready to be released.
Just be patient 🙄

**Comment by Larpoux** [comment]
@hyochan : This request will be fulfilled by Flauto.
You will be allowed to close this issue when Flauto will be released (I home soon 😺 )

**Comment by hyochan** [comment]
Closes via #243
